network_observability/installing-operators.adoc

```diff
@@ -27,16 +27,17 @@ include::modules/network-observability-loki-secret.adoc[leveloffset=+2]
 * For more information about the option to use different namespaces for the separate components, see the `spec.loki.tls.caCert.namespace` specification in the xref:../network_observability/flowcollector-api.adoc#network-observability-flowcollector-api-specifications_network_observability[Flow Collector API Reference] and callout number 5 in the xref:../network_observability/configuring-operator.adoc#network-observability-flowcollector-view_network_observability[Flow Collector sample resource].
 
 include::modules/network-observability-lokistack-create.adoc[leveloffset=+2]
+
 [role="_additional-resources"]
 .Additional resources
 * xref:../logging/cluster-logging-loki.adoc#logging-creating-new-group-cluster-admin-user-role_cluster-logging-loki[Creating a new group for the cluster-admin user role]
 
+include::modules/loki-deployment-sizing.adoc[leveloffset=+3]
```

Comment:
Just want to make sure you intended to offset this module this much. Because of the depth in the assembly, the title does not appear in the mini TOC. [1] IMO it would work perfectly fine with an offset of 2, but totally your call.

[1] https://67435--docspreview.netlify.app/openshift-enterprise/latest/network_observability/installing-operators#loki-deployment-sizing_network_observability

Reply:
@mjpytlak This is a great observation. It didn't really jump out at me honestly because in the live docs, this information currently doesn't appear in the TOC due to the fact that its just a table in an assembly. Now that @abrennan89 made it a module it _can_ display in the TOC. I agree with making it a level 2 so it will be displayed. I think sometimes customers are just looking for this reference and it would be nice to show in the mini TOC.